x/supplier/client/cli/tx_stake_supplier_test.go

```diff
@@ -38,135 +39,205 @@ func TestCLI_StakeSupplier(t *testing.T) {
 		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(net.Config.BondDenom, sdkmath.NewInt(10))).String()),
 	}
 
+	defaultConfig := `
+		- service_id: svc1
+		  endpoints:
+		  - url: http://pokt.network:8081
+		    rpc_type: json_rpc
+		`
+
 	tests := []struct {
-		desc           string
-		address        string
-		stakeString    string
-		servicesString string
-		err            *sdkerrors.Error
+		desc        string
+		address     string
+		stakeString string
+		config      string
+		err         *sdkerrors.Error
 	}{
 		// Happy Paths
 		{
-			desc:           "stake supplier: valid",
-			address:        supplierAccount.Address.String(),
-			stakeString:    "1000upokt",
-			servicesString: "svc1;http://pokt.network:8081",
+			desc:        "stake supplier: valid",
+			address:     supplierAccount.Address.String(),
+			stakeString: "1000upokt",
+			config:      defaultConfig,
 		},
 
 		// Error Paths - Address Related
 		{
 			desc: "stake supplier: missing address",
 			// address:     "explicitly missing",
-			stakeString:    "1000upokt",
-			servicesString: "svc1;http://pokt.network:8081",
-			err:            types.ErrSupplierInvalidAddress,
+			err:         types.ErrSupplierInvalidAddress,
+			stakeString: "1000upokt",
+			config:      defaultConfig,
 		},
 		{
-			desc:           "stake supplier: invalid address",
-			address:        "invalid",
-			stakeString:    "1000upokt",
-			servicesString: "svc1;http://pokt.network:8081",
-			err:            types.ErrSupplierInvalidAddress,
+			desc:        "stake supplier: invalid address",
+			address:     "invalid",
+			stakeString: "1000upokt",
+			err:         types.ErrSupplierInvalidAddress,
+			config:      defaultConfig,
 		},
 
 		// Error Paths - Stake Related
 		{
 			desc:    "stake supplier: missing stake",
 			address: supplierAccount.Address.String(),
-			// stakeString: "explicitly missing",
-			servicesString: "svc1;http://pokt.network:8081",
-			err:            types.ErrSupplierInvalidStake,
+			err:     types.ErrSupplierInvalidStake,
+			// stakeString:    "explicitly missing",
+			config: `
+				- service_id: svc1
+				  endpoints:
+				  - url: http://pokt.network:8081
+				    rpc_type: json_rpc
+				`,
 		},
 		{
-			desc:           "stake supplier: invalid stake denom",
-			address:        supplierAccount.Address.String(),
-			stakeString:    "1000invalid",
-			servicesString: "svc1;http://pokt.network:8081",
-			err:            types.ErrSupplierInvalidStake,
+			desc:        "stake supplier: invalid stake denom",
+			address:     supplierAccount.Address.String(),
+			err:         types.ErrSupplierInvalidStake,
+			stakeString: "1000invalid",
+			config: `
+				- service_id: svc1
```

Comment:
The indentation here seems to be different from the others.

![Screenshot 2023-11-16 at 3 34 07 PM](https://github.com/pokt-network/poktroll/assets/1892194/8507f4d7-bae2-4da9-944d-075870233108)